docs/reference/search/profile.asciidoc

```diff
@@ -216,6 +216,12 @@ a `query` array and a `collector` array.  Alongside the `search` object is an `a
 
 There will also be a `rewrite` metric showing the total time spent rewriting the query (in nanoseconds).
 
+==== Human readable output
+
+As with other statistics apis, the Profile API supports human readable outputs for the time values. Those can be turned on by adding 
+`?human=true` to the query string. In this case, in addition to the `"time_in_nanos"` field, the output contains the additional `"time"`
+field with the time in milliseconds, (e.g. `"time": "0.3919430000ms"`). 
```

Comment:
Yes, after the last change (using XContentHelper#timeValueField) this can be other units too, and the format will not be old the fixed 10-digits. I will update this accordingly.